packages/aws-cdk-lib/aws-events-targets/test/ecs/event-rule-target.test.ts

```diff
@@ -1193,3 +1193,150 @@ test('Imported task definition with revision uses original arn for policy resour
   const template = Template.fromStack(stack);
   template.hasResource('AWS::IAM::Policy', { Properties: policyMatch });
 });
+
+test('throws an error when ephemeral storage size is less than 20 GiB', () => {
```

Comment:
nit: can you use [test.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout) to combine this and the `throws an error when ephemeral storage size is greater than 200 GiB` tests